### PR TITLE
[Linaro:ARM_CI] Add clang support to aarch64 toolchain

### PR DESCRIPTION
**Pull request by elfringham**
Add the ability to use clang instead of gcc in the aarch64 toolchain

### CODE REVIEW

File: tensorflow/tools/toolchains/cpus/aarch64/aarch64_compiler_configure.bzl [hunk 1]
```diff
@@ -55,7 +55,44 @@ def aarch64_compiler_configure():
...
+            "TF_NEED_TENSORRT": "0",
+            "TF_SYSROOT": "/dt10",
+            "TF_TENSORRT_VERSION": "",
+        },
+    )
+
+    ml2014_tf_aarch64_configs(
+        name_container_map = {
+            "ml2014_clang_aarch64": "docker://localhost/tensorflow-build-aarch64",
+            "ml2014_clang_aarch64-python3.7": "docker://localhost/tensorflow-build-aarch64:latest-python3.7",
```
Comment (MichaelHudgins) [inline tensorflow/tools/toolchains/cpus/aarch64/aarch64_compiler_configure.bzl:75]: Should 3.7 be removed and a 3.11 added?

Comment (elfringham) [reply tensorflow/tools/toolchains/cpus/aarch64/aarch64_compiler_configure.bzl:75]: Ack.

**Review by MichaelHudgins — APPROVED** [review]

**Comment by gbaned** [comment]
Hi @MichaelHudgins Can you please review this PR ? Thank you!

**Review by MichaelHudgins — APPROVED** [review]

**Comment by MichaelHudgins** [comment]
@elfringham let me know when you want to get this merged.

**Comment by gbaned** [comment]
Hi @elfringham Any update on this CL? Please. Thank you!

**Comment by elfringham** [comment]
@MichaelHudgins We would like to have this PR merged now please. It adds the toolchain but does not use it by default. This will allow use and testing by other CI systems such as Linaro and ARM.

**Review by MichaelHudgins — APPROVED** [review]

**Comment by MichaelHudgins** [comment]
Sorry on the delay, was OOO last week.  Can you please rebase?

**Review by MichaelHudgins — APPROVED** [review]

**Comment by elfringham** [comment]
@MichaelHudgins I cannot see what the failure on the feedback/copybara check is about. Is it something I need to fix?

**Comment by MichaelHudgins** [comment]
Its from a broken windows build from the last LLVM integrate, so not on your end.

**Pull request merged**